### PR TITLE
Add readme/gitattributes

### PR DESCRIPTION
**Pull request by kaanHapptiq**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+*.ts text eol=lf
+*.css text eol=lf
+*.html text eol=lf
+*.md text eol=lf
+*.sh text eol=lf
+*.xml text eol=lf
+*.json text eol=lf
+*.yml text eol=lf
+*.yaml text eol=lf
+
```
Comment (dkleuser) [inline .gitattributes:19]: Let's add some more Python specifics:
```suggestion
*.pxd    text diff=python eol=lf
*.py     text diff=python eol=lf
*.py3    text diff=python eol=lf
*.pyw    text diff=python eol=lf
*.pyx    text diff=python eol=lf
*.pyz    text diff=python eol=lf
*.pyi    text diff=python eol=lf

*.db     binary
*.p      binary
*.pkl    binary
*.pickle binary
*.pyc    binary export-ignore
*.pyo    binary export-ignore
*.pyd    binary
```

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,42 @@
+*.pbxproj binary merge=union
+
+# Consistent line endings for text files
+*.c text
+*.cpp text
+*.h text
+*.py text eol=lf
```
Comment (dkleuser) [inline .gitattributes:7]: Either remove it in favour of the suggestion I posted below or add the `diff=python` here and remove the line for .py from my suggestions.

File: README.md [hunk 3]
```diff
@@ -1 +1,19 @@
-# happtiq-oss-python-libs
\ No newline at end of file
+# happtiq-oss-python-libs
+
+## Sub-Packages
```
Comment (mab-happtiq) [inline README.md:3]:
```suggestion
## Packages
```

File: README.md [hunk 4]
```diff
@@ -1 +1,19 @@
...
+
+This project is composed of two sub-packages, each providing specific functionalities to enhance your applications. Below are links to the respective sub-packages:
+
+### 1. [happtiq_commons_gen_ai](https://pypi.org/project/happtiq_commons_gen_ai/)
+`happtiq_commons_gen_ai` provides an interface to interact with different AI models like ChatGPT and Gemini. It includes methods to send prompts and manage responses.
+
+[Read the README.md](https://github.com/happtiq/happtiq-oss-python-libs/blob/main/packages/happtiq_commons_gen_ai/README.md)
```
Comment (mab-happtiq) [inline README.md:10]: delete

File: README.md [hunk 5]
```diff
@@ -1 +1,19 @@
...
+[Learn more about happtiq_commons_gen_ai](https://github.com/happtiq/happtiq-oss-python-libs/tree/main/packages/happtiq_commons_gen_ai) (Link to documentation or GitHub repository).
+
+### 2. [happtiq_commons_google_cloud](https://pypi.org/project/happtiq_commons_google_cloud/)
+`happtiq_commons_google_cloud` provides utilities and services to interact with various Google Cloud Platform services, including OAuth2 authentication, Google Cloud Storage (GCS) operations, Pub/Sub messaging, Secret Manager, and more.
+
+[Read the README.md](https://github.com/happtiq/happtiq-oss-python-libs/blob/main/packages/happtiq_commons_google_cloud/README.md)
```
Comment (mab-happtiq) [inline README.md:16]: delete

File: README.md [hunk 6]
```diff
@@ -1 +1,19 @@
-# happtiq-oss-python-libs
\ No newline at end of file
+# happtiq-oss-python-libs
+
+## Sub-Packages
+
+This project is composed of two sub-packages, each providing specific functionalities to enhance your applications. Below are links to the respective sub-packages:
+
+### 1. [happtiq_commons_gen_ai](https://pypi.org/project/happtiq_commons_gen_ai/)
+`happtiq_commons_gen_ai` provides an interface to interact with different AI models like ChatGPT and Gemini. It includes methods to send prompts and manage responses.
```
Comment (mab-happtiq) [inline README.md:8]: delete

File: README.md [hunk 7]
```diff
@@ -1 +1,19 @@
...
+
+[Read the README.md](https://github.com/happtiq/happtiq-oss-python-libs/blob/main/packages/happtiq_commons_gen_ai/README.md)
+[Learn more about happtiq_commons_gen_ai](https://github.com/happtiq/happtiq-oss-python-libs/tree/main/packages/happtiq_commons_gen_ai) (Link to documentation or GitHub repository).
+
+### 2. [happtiq_commons_google_cloud](https://pypi.org/project/happtiq_commons_google_cloud/)
+`happtiq_commons_google_cloud` provides utilities and services to interact with various Google Cloud Platform services, including OAuth2 authentication, Google Cloud Storage (GCS) operations, Pub/Sub messaging, Secret Manager, and more.
```
Comment (mab-happtiq) [inline README.md:14]: delete

File: README.md [hunk 8]
```diff
@@ -1 +1,19 @@
-# happtiq-oss-python-libs
\ No newline at end of file
+# happtiq-oss-python-libs
+
+## Sub-Packages
+
+This project is composed of two sub-packages, each providing specific functionalities to enhance your applications. Below are links to the respective sub-packages:
```
Comment (mab-happtiq) [inline README.md:5]:
```suggestion
Below are links to the different packages:
```

**Review by mab-happtiq — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (kaanHapptiq) [reply README.md:3]: done in the branch

File: .github/workflows/publish_pypi.yml [hunk 9]
```diff
@@ -12,6 +12,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v4
+      with:
```
Comment (mab-happtiq) [inline .github/workflows/publish_pypi.yml:15]: revert

File: README.md [hunk 10]
```diff
@@ -1 +1,9 @@
-# happtiq-oss-python-libs
\ No newline at end of file
+# happtiq-oss-python-libs
+
+## Packages
+
+Below are links to the different packages:
+
+[Learn more about happtiq_commons_gen_ai](https://github.com/happtiq/happtiq-oss-python-libs/tree/main/packages/happtiq_commons_gen_ai) (Link to documentation or GitHub repository).
```
Comment (mab-happtiq) [inline README.md:7]:
```suggestion
[happtiq_commons_gen_ai](https://github.com/happtiq/happtiq-oss-python-libs/tree/main/packages/happtiq_commons_gen_ai)
```

File: README.md [hunk 11]
```diff
@@ -1 +1,9 @@
...
+
+[Learn more about happtiq_commons_google_cloud](https://github.com/happtiq/happtiq-oss-python-libs/tree/main/packages/happtiq_commons_google_cloud) (Link to documentation or GitHub repository).
```
Comment (mab-happtiq) [inline README.md:9]:
```suggestion
[happtiq_commons_google_cloud](https://github.com/happtiq/happtiq-oss-python-libs/tree/main/packages/happtiq_commons_google_cloud)
```

**Review by mab-happtiq — CHANGES_REQUESTED** [review]

**Review by dkleuser — APPROVED** [review]

**Review by mab-happtiq — APPROVED** [review]

**Pull request merged**